packages/website/src/content/docs/changelog/edge.mdx

```diff
@@ -537,21 +537,21 @@ will now deduplicate images that are added to the cache by [kube\_node\_image\_c
 us to install rules onto the cluster to automatically generate, mutate, or validate resources based on a powerful, Kubernetes-native expression
 language. This provides several benefits:
 
-* Provides a unified control plane for adding functionality that previously required managing additional controllers or custom scripts.
-* Allows us to simplify several parts of our IaC modules by offloading resource management to global Kyverno policies.
-* Allows us to add Panfactum-compatible, sensible defaults to Kubernetes resources that are not created by Panfactum modules.
-* Allows users to add management logic to their clusters that was previously only possible by building and deploying custom controllers. See the [example policies.](https://kyverno.io/policies/)
+  * Provides a unified control plane for adding functionality that previously required managing additional controllers or custom scripts.
```

Comment:
?

Reply:
@fullykubed I'm not sure if it's our lint or the build process but these should be indented. There are more instances of this that make reading the changelog difficult right now. 

reverted here but created an issue for it.

Reply:
https://github.com/Panfactum/stack/issues/316